app/src/main/java/org/mozilla/fenix/share/ShareViewModel.kt

```diff
@@ -89,12 +92,18 @@ class ShareViewModel(application: Application) : AndroidViewModel(application) {
                 type = "text/plain"
                 flags = Intent.FLAG_ACTIVITY_NEW_TASK
             }
-            val shareAppsActivities = getIntentActivities(shareIntent, getApplication())
-            var apps = buildAppsList(shareAppsActivities, getApplication())
+            val shareAppsActivities = getIntentActivities(shareIntent, context)
+
+            var apps = buildAppsList(shareAppsActivities, context)
             recentAppsStorage.updateDatabaseWithNewApps(apps.map { app -> app.activityName })
             val recentApps = buildRecentAppsList(apps)
             apps = filterOutRecentApps(apps, recentApps)
 
+            /* if copy app is available, prepend to the list of actions */
```

Comment:
```suggestion
            // if copy app is available, prepend to the list of actions
```

😛

Reply:
Kidding of course, don't change this unless there is a real reason to edit this patch.